scalene/scalene_profiler.py

```diff
@@ -1562,15 +1502,18 @@ def main() -> None:
                         exit_status = 0
                         profiler.start()
                         # Run the code being profiled.
+                        import sys as _sys
+                        _sys.argv = ['abcde']
+                        globals = dict(the_globals)
+                        globals['sys'] = _sys
                         try:
                             exec(code, the_globals, the_locals)
                         except SystemExit as se:
                             # Intercept sys.exit and propagate the error code.
                             exit_status = se.code
                         except BaseException:
-                            if Scalene.__debug:
-                                print(traceback.format_exc())  # for debugging only
-                            pass
+                            print(traceback.format_exc())  # for debugging only
```

Comment:
Mentioned for previous PR; should probably just do whatever Python does by default, which may not _exactly_ be this.

Reply:
Closed previous PR to consolidate-- this appears to be the default behavior, but I'll verify in CPython